packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -65,6 +69,29 @@ export function TableNG(props: TableNGProps) {
   const [isInspecting, setIsInspecting] = useState(false);
   const [isContextMenuOpen, setIsContextMenuOpen] = useState(false);
 
+  const headerCellRefs = useRef<Record<string, HTMLDivElement>>({});
+  const [readyForRowHeightCalc, setReadyForRowHeightCalc] = useState(false);
+
+  // This state will trigger re-render for recalculating row heights
+  const [resizeTrigger, setResizeTrigger] = useState(0);
```

Comment:
Neither of these states are being read anywhere? Are they just being used to force re-render?

Reply:
yes, this is only for making react-data-grid to re-calculate rowHeight

Reply:
I cleaned up a bit, just removed those state variables. There is definitely space for improvement, how we update our table.